### ISSUE TITLE
Fix broken tests with update poetry.lock

### ISSUE THREAD

**Issue by dusktreader** [dusktreader/flask-praetorian#266]
```
cls = <class 'freezegun.api.FakeDatetime'>, tz = datetime.timezone.utc

    @classmethod
    def now(cls, tz=None):
        now = cls._time_to_freeze() or real_datetime.now()
        if tz:
>           result = tz.fromutc(now.replace(tzinfo=tz)) + cls._tz_offset()
E           ValueError: fromutc: dt.tzinfo is not self

../../../.cache/pypoetry/virtualenvs/flask-praetorian-IlIQAbrd-py3.8/lib/python3.8/site-packages/freezegun/api.py:358: ValueError
```

**Comment by dusktreader** [comment]
Found this: https://github.com/spulec/freezegun/issues/411

Solution will be to drop freezegun

**Comment by jvllmr** [comment]
This issue is caused by an issue with pyJWT 2.2.0, see PR #264 and PR #263 for this. You can test this behaviour by bumping only pyJWT below 2.2.0 again.

**Comment by dusktreader** [comment]
I've been wanting to remove freezegun for a while, though. I use pendulum in the tests because it massively improves readability and ease of manipulating timestamps. I'm working on a solution that will give us freezegun like functionality with pendulum and will have a PR that fixes this issue soon.

**Comment by dusktreader** [comment]
resolved in PR #268
